internal/gengapic/gengapic_test.go

```diff
@@ -485,11 +493,14 @@ methods:
 	} {
 		g.reset()
 		g.descInfo.ParentElement[tst.m] = serv
+		serv.Method = []*descriptor.MethodDescriptorProto{
+			tst.m,
+		}
```

Comment:
nit: add key

Reply:
This is the inline creation of a slice with a single element. It's very confusing, but `ServiceDescriptorProto.Method` is a collection of `[]*MethodDescriptorProto`. It should have been named `Methods`, right?